storage/kvrocks.conf

```diff
@@ -849,3 +849,4 @@ rocksdb.rate_limiter_auto_tuned yes
 
 ################################ NAMESPACE #####################################
 # namespace.test change.me
+backup-dir .//backup
```

Comment:
I need to make sure it's not causing any issues on existing installs, coule you also remove it for now?